.typespec/uploads/operations.tsp

```diff
@@ -24,7 +24,11 @@ interface Uploads {
     
     For guidance on the proper filename extensions for each purpose, please follow the documentation on [creating a File](/docs/api-reference/files/create).
     """)
-  createUpload(@body requestBody: CreateUploadRequest): Upload | ErrorResponse;
+  createUpload(
+    @header accept: "application/json",
```

Comment:
oh, wow... I know how much of a pain this must have been. Thank you for slogging through all these boilerplate updates!